kubernetes-scenarios/scenario-1/step2.md

```diff
@@ -10,7 +10,7 @@ And:
 
 `kubectl get events --sort-by=.metadata.creationTimestamp`{{execute}}
 
-The first command, as it suggests describes the pods in the namespace, if you wanted to describe a single pod you'd simply add the pod name to the end of the command `kubectl describe pods REPLACE-WITH-POD-NAME`. The seconds command returns all the recent events for the namespace you're in.
+The first command, as it suggests describes the pods in the namespace, if you wanted to describe a single pod you'd simply add the pod name to the end of the command `kubectl describe pods REPLACE-WITH-POD-NAME`{{execute}}. The seconds command returns all the recent events for the namespace you're in.
```

Comment:
The reason why it's not executable is that this will produce an error, it was more to explain you could run describe with a specific pod name